src/dymaptic.Blazor.GIS.API.Core/Components/Views/MapView.razor.cs

```diff
@@ -431,12 +431,23 @@ protected override async Task OnParametersSetAsync()
 
     protected override async Task OnAfterRenderAsync(bool firstRender)
     {
-        ViewJsModule = await JsRuntime
-            .InvokeAsync<IJSObjectReference>("import",
-                "./_content/dymaptic.Blazor.GIS.API.Core/arcGisJsInterop.js");
-
         if (firstRender)
         {
+            try
```

Comment:
Does this cause an extra round trip to the server by first checking if the JS file can be downloaded?

Per our other discussion where this came up, @TimPurdum suggested an assembly check or an `IConfiguration` value. I think I like the assembly check idea.